xml/System.IO/File.xml

```diff
@@ -3198,7 +3198,13 @@ The following example moves a file.
         <exception cref="T:System.ArgumentNullException">
           <paramref name="sourceFileName" /> or <paramref name="destFileName" /> is <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentException">.NET Framework and .NET Core versions older than 2.1: <paramref name="sourceFileName" /> or <paramref name="destFileName" /> is a zero-length string, contains only white space, or contains invalid characters. You can query for invalid characters by using the <see cref="M:System.IO.Path.GetInvalidPathChars" /> method.</exception>
-        <exception cref="T:System.UnauthorizedAccessException">The caller does not have the required permission.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+        The caller does not have the required permission.
+      
+        -or-
+
+        The Operating System has failed to acquire an exclusive access to the destination file.
+        </exception>
```

Comment:
Subsequent lines need to be indented with one space only. See [this exception ](https://github.com/dotnet/dotnet-api-docs/blob/4c6279f88d8411618554ff3641cfe38fedddc630/xml/System.IO/File.xml#L3567-L3580)as an example.

```suggestion
        <exception cref="T:System.UnauthorizedAccessException">The caller does not have the required permission.
 
 -or-
 
 The Operating System has failed to acquire an exclusive access to the destination file.</exception>
```